apps/src/dance/DanceVisualizationColumn.jsx

```diff
@@ -11,7 +11,7 @@ import i18n from '@cdo/locale';
 import AgeDialog from '../templates/AgeDialog';
 import {getFilteredSongKeys, getFilterStatus} from '@cdo/apps/dance/songs';
 
-const SongSelector = Radium(
+export const SongSelector = Radium(
```

Comment:
Yep! That's a separate task [here](https://codedotorg.atlassian.net/browse/LABS-164) (looks like @AfifahK will be working on it)

Reply:
I was planning for DanceView and DanceVisualizationColumn to be different - I want to avoid using some of the legacy components like `ProtectedVisualizationDiv` and `GameButtons`

Reply:
Cool - sounds good!